### DISCUSSION TITLE
suggestion for the 6 slot adapter

### DISCUSSION THREAD

**Discussion by mer2329** [sanni/cartreader#569] [Ideas & Suggestions]
i would like to suggest 2 changes to the n64 controller connector on the 6 slot adapter. 
1) put another hole for the resistor as its a little awkward to solder that and a wire to. (the hole for dat has a resistor that goes into the same hole as the wire)
2) change the pitch for the 3 wires on the controller port from 3mm to 2.5 so normal header pins or a JST header can be used

i have no kicad skills (i just cant seem to wrap my head around component libraries, i keep getting thrown for a loop) so this is all i can do.

**Comment by sanni** [comment]
Done.

**Comment by mer2329** [comment]
WOW that was fast. thanks a TON.
